components/infobox/wikis/chess/infobox_league_custom.lua

```diff
@@ -74,7 +78,16 @@ function CustomInjector:parse(id, widgets)
 			Cell{name = 'Restrictions', content = caller:createRestrictionsCell(args.restrictions)}
 		)
 	elseif id == 'gamesettings' then
-		return {Cell{name = 'Variant', content = {caller.data.mode}}}
+		table.insert(
+			widgets,
+			Cell{name = 'Time Control', content = {caller.data.mode}}
+		)
+		if caller.data.game ~= GAMES['chess'] then
+			table.insert(
+				widgets,
+				Cell{name = 'Variant', content = {caller.data.game}}
+			)
+		end
```

Comment:
use Array.append

Reply:
For inserting to a table? Why?

Reply:
```suggestion
		Array.appendWith(widgets,
			Cell{name = 'Time Control', content = {caller.data.mode}},
			caller.data.game ~= GAMES['chess'] and Cell{name = 'Variant', content = {caller.data.game}} or nil
		)
```
(not liking the `GAMES['chess']` as i should use Module:Game here instead)